xml/depl_maintenance.xml

```diff
@@ -565,10 +565,14 @@
     </listitem>
     <listitem>
      <para>
-       If your &productname; 6 setup is configured to use LBaaS v1, you need to
+       If your &productname; 6 setup is using LBaaS v1, you need to
        switch to LBaaS v2 manually. To do this, switch to the
        <systemitem>Raw</systemitem> view in the &o_netw; &barcl; and set the
-       <systemitem>use_lbaasv2</systemitem> parameter to <literal>true</literal>.
+       <systemitem>use_lbaasv2</systemitem> parameter to
+       <literal>true</literal>. It is also necessary to re-create all the
+       object manually. This includes load balancers pools, and health
```

Comment:
*load balancers, pools*

Just note that users know what they should do: they've created those objects manually before, otherwise there wouldn't be any.